test/controllers/weets.spec.js

```diff
@@ -121,4 +122,127 @@ describe('Module controllers', () => {
       });
     });
   });
+
+  describe('GET weets', () => {
+    describe('Successfull cases', () => {
+      let response = {};
+      beforeAll(async () => {
+        successDecode({ email: 'fake@domain.com' });
+        await create({ ...user(), email: 'fake@domain.com' });
+        const weetsQuantity = 10;
+        for (let i = 0; i < weetsQuantity; i++) {
+          successDecode({ email: 'fake@domain.com' });
+          await mockSuccessRequest('This is a random fact');
+          await getResponse({
+            endpoint: '/weets',
+            method: 'post',
+            header: { authorization: 'Bearer token' }
+          });
+        }
+      });
+      afterAll(() => truncateDatabase());
+      describe('Get weets successfully', () => {
+        afterAll(() => truncateDatabase());
+        it('The response status code must be 200', async () => {
+          successDecode({ email: 'fake@domain.com' });
```

Comment:
`successDecode` and  `response = await requestWeetsByPages(2, 1)` must called in beforeAll

Reply:
Remember that